lib/getAllPosts.ts

```diff
@@ -0,0 +1,16 @@
+import { toDual } from './data';
+import { ApiPost, DualPost } from './types';
+
+const bucketSlug = process.env.NEXT_PUBLIC_COSMIC_BUCKET_SLUG;
+const readKey = process.env.NEXT_PUBLIC_COSMIC_READ_KEY;
```

Comment:
i think we need to think carefully about whether these should be prefixed with `NEXT_PUBLIC_` or not. not that there's any great security concern, but these public env vars are for things that will actually get built into the bundled javascript. and this function should only run at build time (i think), and thus, it should be able to use actual secrets.